website/common/locales/en/gear.json

```diff
@@ -101,7 +102,7 @@
   "weaponSpecialSkiText": "Ski-sassin Pole",
   "weaponSpecialSkiNotes": "A weapon capable of destroying hordes of enemies! It also helps the user make very nice parallel turns. Increases Strength by <%= str %>. Limited Edition 2013-2014 Winter Gear.",
   "weaponSpecialCandycaneText": "Candy Cane Staff",
-  "weaponSpecialCandycaneNotes": "A powerful mage's staff. Powerfully DELICIOUS, we mean! Two-handed weapon. Increases Intelligence by <%= int %> and Perception by <%= per %>. Limited Edition 2013-2014 Winter Gear.",
```

Comment:
We do want "Two-handed weapon." to be removed from that text now, for consistency with all the other two-handed gear. This is the only item that specifies the handedness in its Notes. See the issue for details (https://github.com/HabitRPG/habitica/issues/8441).

The purpose of the issue is to have "Two-handed item."  automatically added to the item descriptions returned by the API, so the mobile apps should pick up that extra text automatically. However @AndrzejDoyle it looks like you've added it in the website's code only.